paket.dependencies

```diff
@@ -1,16 +1,15 @@
-source http://nuget.org/api/v2
+source https://api.nuget.org/v3/index.json
+source https://nuget.org/api/v2
 redirects: on
 
 nuget FSharp.Data
 nuget FAKE
 nuget CommandLineParser
-nuget FSharp.Core
+nuget FSharp.Core 4.0.0.1 redirects:force
```

Comment:
there were some binding redirect issues that popped up like NUnit was the problem initially

```
NUGET
  remote: https://api.nuget.org/v3/index.json
    CommandLineParser (1.9.71)
    FAKE (4.56)
    FSharp.Compiler.Service (2.0.0.6)
    FSharp.Core (4.0.0.1) - redirects: force
    FSharp.Data (2.3.2)
```

I didn't upgrade FCS though